Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -94,11 +94,9 @@ private void ProtoTestSingle(string testProtoName, bool ignoreCsProj = false, bo
             // Verify each output file.
             foreach (var file in files)
             {
-                if (ignoreCsProj && file.RelativePath.EndsWith(".csproj"))
-                {
-                    continue;
-                }
-                if (ignoreSnippets && file.RelativePath.Contains($".Snippets{Path.DirectorySeparatorChar}"))
+                if (ignoreCsProj && file.RelativePath.EndsWith(".csproj") ||
```

Comment:
I honestly can't remember precedence rules for `&&` and `||`, but I think the code would at least be clearer with this being explicit:

```csharp
if ((ignoreCsProj && file.RelativePath.EndsWith(".csproj")) ||
    (... && ...) ||
    (... && ...))
```

Reply:
Oops, yes, extra parentheses added.

Reply:
And something you can't remember about C#! Incredible!!